src/NServiceBus.Transport.RabbitMQ/Receiving/MessagePumpConnectionFailedCircuitBreaker.cs

```diff
@@ -59,7 +59,7 @@ void CircuitBreakerTriggered(object state)
         string name;
         TimeSpan timeToWaitBeforeTriggering;
         Timer timer;
-        Action<Exception> criticalErrorAction;
+        Action<string, Exception> criticalErrorAction;
```

Comment:
This class doesn't need to change. These changes revert it back to how it was before https://github.com/Particular/NServiceBus.RabbitMQ/pull/763